### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [TeamLetsGetIt/OurIdea#1]
Great job creating your organization and app design repo! 👍 However, it looks like you're missing the required appideas.md file or have not added the content from your benchmark and brainstorm session. Please see step 3 in the Group Activity Tab for more instructions. Once you've updated this file, please re-submit by clicking the Submit button on the Group Activity Tab for the associated Group Activity.

**Comment by codepathreview** [comment]
Nice work! ?? In this activity, you examined your favorite production apps to benchmark and gain insight into elements that makeup great apps. Did you find similar elements when examining your group's new app ideas? In our next Group App Design Activity, you'll want to take this into consideration as you finalize the app idea your group will design.
 
/cc @codepathreview

**Comment by bydavid** [comment]
Yes we did. Our understanding was that we replace our original readme with all the other app reviews with this one?
Also we received an email that we were missing a component of our deliverables? Might we get some clarification on that?

________________________________

**Comment by bydavid** [comment]
I've resubmitted our original ideas and app evaluations in a readme called "appideas.md"
Thanks!

**Comment by codepathreview** [comment]
Hi David,
Please update your README file by updating the link of your complete pitch presentation.
Thanks 
/cc @codepathreview

**Comment by bydavid** [comment]
I updated it!


________________________________
